targets/liteos-stm32f4/source/main.c

```diff
@@ -0,0 +1,123 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "los_sys.h"
+#include "los_tick.h"
+#include "los_task.ph"
+#include "los_config.h"
+#include "los_bsp_led.h"
+#include "los_bsp_key.h"
+#include "los_bsp_uart.h"
+#include "los_bsp_adapter.h"
+
+#include "stm32f429i_discovery.h"
+#include <string.h>
+#include "jerryscript.h"
+#include "jerry_run.h"
+
+static UINT32 g_uwboadTaskID;
+LITE_OS_SEC_TEXT VOID LOS_BoadExampleTskfunc(VOID)
+{
+    while (1)
+    {
+	run();
+    }
+}
+void LOS_BoadExampleEntry(void)
```

Comment:
Separate from previous function with an empty line.

Boad or Board? (BTW, is it board-specific at all?)

(LOS_ ?)